pkg/api/helpers.go

```diff
@@ -178,3 +179,53 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	}
 	return c
 }
+
+// GetAccessModesAsString returns a string representation of an array of access modes.
+// modes, when present, are always in the same order: RWO,ROX,RWX.
+func GetAccessModesAsString(modes []PersistentVolumeAccessMode) string {
+	modesAsString := ""
+
+	if contains(modes, ReadWriteOnce) {
+		appendAccessMode(&modesAsString, "RWO")
```

Comment:
The order would not be determinant if I used a map.   The string is conveniently in ascending order of access.